app/views/admin/tenants/index.html.erb

```diff
@@ -1,21 +1,72 @@
 <div id="tenants_header">
   <h1>Tenants</h1>
```

Comment:
Personally, I would leave the paths...I just think front-facing things should make sense to the users.